capitains.rng

```diff
@@ -139,25 +186,144 @@
       <text/>
     </element>
   </define>
-  
+  <define name="dc.contributor">
+    <element name="dc:contributor">
+      <a:documentation xml:lang="eng"> This documents hold contributor informations according to the
```

Comment:
"documents hold" doesn't make sense to me here. I would think it should be either "This documents" or "This holds", but not both.
This goes for the other new documentation strings as well.

Reply:
Also, "informations" should be "information". For whatever reason this is always a singular noun in English.

Reply:
@sonofmun could you actually correct them ? I am fine with you appearing in the git log ;)